docs/farming-&-staking/staking/operators.mdx

```diff
@@ -262,22 +262,27 @@ services:
       - "0.0.0.0:30433:30433/tcp"
     restart: unless-stopped
     command: [
-# Make sure to replace your_domain_id and your_operator_id with your domain ID and operator ID
-# Adjust the keystore path to the location where you generated your operator keypair.
       "--chain", "gemini-3g",
-      "-- ",
-      "--domain-id", "your_domain_id",
-      "--operator-id", "your_operator_id",
-      "--bootnodes", "/ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp",
-      "--bootnodes", "/ip4/140.82.45.36/tcp/40333/p2p/12D3KooWJxB2PVzyNqZywoCWAMLGhKr2LCbFMXbAPzUPQZon7yt1",
-      "--bootnodes", "/ip4/167.179.116.135/tcp/40333/p2p/12D3KooWHsRae3ip2uQAUPtpMcStHGGw777SJT3nu9maFmtsGry7",
+      "--base-path", "/var/subspace",
+      "--port", "30333",
+      "--dsn-listen-on", "/ip4/0.0.0.0/udp/30433/quic-v1",
+      "--dsn-listen-on", "/ip4/0.0.0.0/tcp/30433",
       "--listen-addr", "/ip4/0.0.0.0/tcp/40333",
       "--rpc-cors", "all",
       "--rpc-methods", "unsafe",
       "--rpc-external",
+      # Replace INSERT_YOUR_ID with your node ID (will be shown in telemetry)
+      "--name", "INSERT_YOUR_ID",
+      "--",
+      # Replace INSERT_YOUR_DOMAIN_ID with domain ID you want to be operator on
+      "--domain-id", "INSERT_YOUR_DOMAIN_ID",
+      "--chain", "gemini-3g",
+      # Replace INSERT_YOUR_OPERATOR_ID with your operator ID
+      "--operator-id", "INSERT_YOUR_OPERATOR_ID",
+      "--bootnodes", "/ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp",
+      "--bootnodes", "/ip4/140.82.45.36/tcp/40333/p2p/12D3KooWJxB2PVzyNqZywoCWAMLGhKr2LCbFMXbAPzUPQZon7yt1",
+      "--bootnodes", "/ip4/167.179.116.135/tcp/40333/p2p/12D3KooWHsRae3ip2uQAUPtpMcStHGGw777SJT3nu9maFmtsGry7",
       "--keystore-path", "/keystore"
```

Comment:
We will need to mount this folder in the `volumes` section. Something like `- /path/to/keystore:/keystore:ro`.